coordinator/charmcraft.yaml

```diff
@@ -127,6 +140,21 @@ requires:
     interface: catalogue
     description: |
       Integration to help users discover Tempo's UI, providing visibility into Tempo's cluster members and their health status.
+  service-mesh:
+    limit: 1
+    interface: service_mesh
+    description: |
+      Subscribe this charm into a service mesh and create authorization policies.
+      We forward to the beacon our authorization policies.
+      The beacon sends the pod and service labels required by this charm to join the mesh.
+      This relation is a pre-requisite for using the provide-cmr-mesh relation.
+  require-cmr-mesh:
+    # TODO: remove this relation when this is fixed:
+    #   https://github.com/canonical/istio-beacon-k8s-operator/issues/91
```

Comment:
This is not nice, because it means we'll have to make a breaking change by removing an endpoint we already now know we won't need?
Can we prioritise fixing the bug instead of adding this?